rsub.sh

```diff
@@ -12,62 +37,73 @@ usage() {
 	exit 1
 }
 
-trap '' HUP
-append=1
+init() {
+	if ! check_absolute_path "$target"; then
+		>&2 echo "rsub: $target is not an absolute path"
+		exit 1
+	fi
 
-while getopts Al:r: arg; do
-	case "$arg" in
-		A) append=0 ;;
-		r) line_regex="$OPTARG" ;;
-		l) line_text="$OPTARG" ;;
-		?) usage ;;
-	esac
-done
-shift $(($OPTIND - 1))
-[ $# -eq 1 ] || usage
+	test -f "$target" || {
```

Comment:
Just wonder why it is not `[` as in other conditions...

Reply:
Good observation! I'll replace `test` with `[` since this is the pattern in `rinstall`